force-app/main/default/classes/ListDonation.cls

```diff
@@ -0,0 +1,144 @@
+public with sharing class ListDonation {
+
+    static final String HARD_CREDIT = 'Hard Credit';
+    static final String SOFT_CREDIT = 'Soft Credit';
+    static final String PARTIAL_SOFT_CREDIT = 'Partial Soft Credit';
+
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description Donation History inner class.
+    */
+    public class DonationHistory{
+        public Id id { get; set; }
+        public String name { get; set; }
+        public Date closeDate { get; set; }
+        public String role { get; set; }
+        public String creditType { get; set; }
+        public Decimal amount { get; set; }
+        //public String paymentMethod { get; set; }
+    }
+
+    /** 
+    * @author Salesforce.org
+    * @date 2021
+    * @description method to return Donation History list.
+    */
+    public static void getDonationHistory(){
```

Comment:
Should this method be using "@AuraEnabled" ?

Reply:
yes, added